build.psm1

```diff
@@ -920,7 +922,24 @@ function Start-PSPester {
         }
         else
         {
-            & $powershell -noprofile -c $Command
+            if ($PassThru.IsPresent)
+            {
+                $passThruFile = [System.IO.Path]::GetTempFileName()
+                try 
+                {
+                    $Command += "|Export-Clixml -Path '$passThruFile' -Force"
+                    Start-NativeExecution -sb {& $powershell -noprofile -c $Command} | ForEach-Object { Write-Host $_}
```

Comment:
Since all output from `Invoke-Pester` will be captured by `Export-Clixml`, what will be passed to `Foreach-Object`?